src/Compression/ZipArchive.class.st

```diff
@@ -271,7 +271,7 @@ ZipArchive >> readFrom: aStreamOrFile [
 	| stream name eocdPosition |
 	stream := aStreamOrFile isStream
 		ifTrue: [ name := aStreamOrFile printString. aStreamOrFile ]
-		ifFalse: [ name := aStreamOrFile asFileReference fullName. aStreamOrFile asFileReference binaryReadStream ].
+		ifFalse: [ name := aStreamOrFile asFileReference fullName. aStreamOrFile asFileReference binaryReadStream contents readStream ].
```

Comment:
Here the `contents` is doing the magic to load the file to memory, right?

Can we add a comment?
```suggestion
		ifFalse: [ name := aStreamOrFile asFileReference fullName. 
			"Force to load file contents"
			aStreamOrFile asFileReference binaryReadStream contents readStream ].
```

Reply:
Thanks! I published a comment inspired in yours but a bit more explicit ^^